doc/design/from_ngsi_events_to_flume_events.md

```diff
@@ -95,4 +104,4 @@ Fermín Galán Márquez (fermin.galanmarquez@telefonica.com) **[Contributor and
 <br>
 Germán Toro del Valle (german.torodelvalle@telefonica.com) **[Contributor]**
 <br>
-Iván Arias León (ivan.ariasleon@telefonica.com) **[Quality Assurance]**
\ No newline at end of file
+NoIván Arias León (ivan.ariasleon@telefonica.com) **[Quality Assurance]**
```

Comment:
LOL :) 

Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/178d84aaea0fbe4befefab431ad9fcd162ebdf9c